server/model/pipeline.go

```diff
@@ -78,3 +129,14 @@ type PipelineOptions struct {
 	Branch    string            `json:"branch"`
 	Variables map[string]string `json:"variables"`
 } //	@name PipelineOptions
+
+type Deployment struct {
+	Target      string `json:"target"`
+	Task        string `json:"task"`
+	Description string `json:"description"`
+}
+
+type Release struct {
+	IsPrerelease bool   `json:"is_prerelease,omitempty"`
+	TagTitle     string `json:"release_tag_title,omitempty"`
```

Comment:
```suggestion
	TagTitle     string `json:"tag_title,omitempty"`
```

Do we want to know the tag title or the release title here?